### ISSUE TITLE
Spread arguments in new call don't work

### ISSUE THREAD

**Issue by yukulele** [zenparsing/esdown#23]
Spread arguments don't work with constructor call

example:
```javascript
let a = [1, 2, ...arr, 4, 5]; //works
Math.max(...arr); //works
myFunction(...arr); //works
new myClass(...arr); //error
```

**Comment by zenparsing** [comment]
Thanks!

Babel supports this by creating a single-use bound function.

**Comment by yukulele** [comment]
like that? 
```javascript
new (myClass.bind.apply(myClass, [null].concat(arr)));
```

**Comment by zenparsing** [comment]
Right

Fixed in 8af8503
